src/util/config.rs

```diff
@@ -23,53 +23,43 @@ use rocksdb::{DBCompressionType, DBRecoveryMode};
 quick_error! {
     #[derive(Debug)]
     pub enum ConfigError {
-        RocksDB
-        ReadableNumber
         Limit(msg: String) {
             description(msg)
             display("{}", msg)
         }
         Address(msg: String) {
             description(msg)
-            display("{}", msg)
+            display("config address error: {}", msg)
         }
         StoreLabels(msg: String) {
             description(msg)
-            display("{}", msg)
+            display("store label error: {}", msg)
+        }
+        Value(msg: String) {
+            description(msg)
+            display("config value error: {}", msg)
         }
     }
 }
 
 pub fn parse_rocksdb_compression(tp: &str) -> Result<DBCompressionType, ConfigError> {
-    match &*tp.to_lowercase() {
+    match &*tp.trim().to_lowercase() {
         "no" => Ok(DBCompressionType::DBNo),
         "snappy" => Ok(DBCompressionType::DBSnappy),
         "zlib" => Ok(DBCompressionType::DBZlib),
         "bzip2" => Ok(DBCompressionType::DBBz2),
         "lz4" => Ok(DBCompressionType::DBLz4),
         "lz4hc" => Ok(DBCompressionType::DBLz4hc),
-        _ => Err(ConfigError::RocksDB),
+        "zstd" => Ok(DBCompressionType::DBZstd),
+        v => Err(ConfigError::Value(format!("invalid compression type {:?}", v))),
```

Comment:
`v` is a temporary variable during processing, using `tp` is more clear and straight forward for user. You can raise a pr to clean up all the other places. But since this pr is about to fix error printing, so it's fine to fix them all in this pr too.